CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## [Unreleased]
+### Fixed
```

Comment:
### Changed
- STI Models - Add a select input with predefined existing STI type values in create/update forms.
- STI Models - Children models do not generate a specific collection anymore (only parent model). 

(something like this)